src/index.tsx

```diff
@@ -44,9 +45,11 @@ class Experiment extends React.Component<ExperimentProps, State> {
   };
 
   updateVariant = (value: string | undefined | null) => {
-    this.setState(() => ({
-      variant: value === undefined || value === null ? "0" : value
-    }));
+    const variant = value === undefined || value === null ? "0" : value;
+    this.setState(
+      () => ({ variant }),
```

Comment:
```suggestion
      { variant },
```

Reply:
I do agree that it will not make a difference here. However, I would like us to be in the habit of always using the function pattern because of this:

https://medium.com/@wisecobbler/using-a-function-in-setstate-instead-of-an-object-1f5cfd6e55d1